projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -167,6 +168,7 @@ if (environment.requestedDeliveryDate) {
     PaymentMethodsModule,
     NotificationPreferenceModule,
     MyInterestsModule,
+    MyaccountViewSideNavigationModule,
```

Comment:
[MAJOR]
1. Do you envision any other my-account-related modules to be created in the near future?
If yes, maybe it would be good to include here only one module (like `MyAccountModule`) which inside would encapsulate all the fine grained modules related to my account feature?

2. Importing the module eagerly in SpartacusFeaturesModule will cause the main.js bundle to increase. In other words, the end-user who visits the homepage will have to load JS code with this module, which at the first sight doesn't make sense.
Did you consider having a separate lazy-loadable feature for my-account related stuff?

Reply:
As discussed in the architecture review, for My Account Landing page my account related modules were asked to be created in the core related modules. Hence it was created as such.

Reply:
> decision was made to maintain each component/module within its respective library
Agreed.

That said, there might be some shared components reused across various my-account pages, like to top left box (with user name and logout link) or the left-hand menu with links. They don't have their respective library yet.

What I'm leading to:
1. have one module for all such generic my-account shared things (the ones that don't belong to any specific library/feature). Thanks to that, we will import only _one_ module related to MyAccount intead of many fine-grained modules in the `SpartacusFeatureModules`

2. Do you think it make sense to lazy load such _shared_ My Account things? Can you foresee that a lot of more other things could go there in the near future (knowing the near-term plans for the Enhanced MyAccount screens)?

Reply:
Hi Kris, as discussed in call today, keeping navigation separately and eager load it